packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -202,7 +202,7 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
           // eslint-disable-next-line react/no-multi-comp
           Header: () => (
             <>
-              <Breadcrumbs />
+              <Breadcrumbs topics={topics} />
```

Comment:
1. we have also breadcrumbs component in sublayout component. You don't need to pass topics there?
2. Wondering why this has to happen in breadcrumbs component? I am not fan of this, as breadcrumbs should be responsible for displaying breadcrumbs, not for validating topics and doing redirects as an effect. Can't we put this checking logic into DiscussionsPage.tsx?

Reply:
@masvelio Yeah, good call. Will change it to be checked in DiscussionsPage.tsx.

Reply:
@masvelio Moved code into DiscussionsPage.tsx.